### ISSUE TITLE
No smoking / no portable devices AUTO doesn't clear the SIGNS landing ECAM message

### ISSUE THREAD

**Issue by Venorcis** [flybywiresim/a32nx#6962]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2022-03-24T17:31:46+00:00",
  "ref": "master",
  "sha": "cdb1a6281a8da541cbce47ff45fd5edf66fccb08",
  "actor": "RichardPilbery",
  "event_name": "manual",
  "pretty_release_name": "master:cdb1a628"
}
```

### Describe the bug

No smoking / no portable devices set to AUTO no longer clears SIGNS the landing ECAM message, e.g. the AUTO setting keeps the "ON" message for "SIGNS" visible; both seatbelts signs and no smoking / no portable devices must be set to ON to clear the message now.

### Expected behavior

Either AUTO or ON should be sufficient (in combination with seatbelts ON) to clear the SIGNS ECAM message upon landing.

### Steps to reproduce

1. Get the plane in landing phase
2. Set seatbelts signs to ON and no smoking / no portable devices to AUTO

### References (optional)

You can see me fiddling trying to clear the blue messages here: https://www.twitch.tv/videos/1435505521?t=01h21m00s

**Comment by beheh** [comment]
Probably caused by #6910.

**Comment by RichardPilbery** [comment]
Thanks for reporting this. Should be fixed by #6967
